infra/helper.py

```diff
@@ -453,9 +455,10 @@ def build_fuzzers_impl(project_name, clean, engine, sanitizer, architecture,
         'bash', '-c', 'cp -r /msan /work'])
     env.append('MSAN_LIBS_PATH=' + '/work/msan')
 
-  command = (
-      ['docker', 'run', '--rm', '-i', '--cap-add', 'SYS_PTRACE'] +
-      _env_to_docker_args(env))
+  command = ['docker', 'run', '--rm']
+  if sys.stdin.isatty():
```

Comment:
@oliverchang WDYT of this?

@Leo-Neat Please comment on why this is necessary.

Reply:
It is necessary to allow for a non-TTY interface to use this function. Github actions is non TTY and requires -i to be removed when running.

Reply:
This seems reasonable to me. I'd change the comment to be a bit more general, e.g. "Support environments with a TTY"

Also, can we just use docker_run here, to keep this logic in a single place? If for some reason docker_run doesn't fit here, let's refactor it so it does.

Reply:
Done.